src/main/java/Task.java

```diff
@@ -0,0 +1,26 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
```

Comment:
I noticed that there is no getter for isDone, and seeing that it is protected, the variable is still accessible by child/parent classes. However for the strongest form of encapsulation, private would do the trick.